### PR TITLE
smp : second core enable, arndale is not supported yet

### PR DESCRIPTION
**Pull request by joohyun0115**
second core enable, arndale is not supported yet

### CODE REVIEW

**Comment by kesl** [comment]
CI : Test PASSed.
Refer to this link for build results: http://keslproject.iptime.org:8082/job/k-hypervisor_pull_request/215/

**Comment by kesl** [comment]
CI : Test PASSed.
Refer to this link for build results: http://keslproject.iptime.org:8082/job/k-hypervisor_pull_request/216/

File: platform-device/cortex_a15x2_arndale/boot.S [hunk 1]
```diff
@@ -160,15 +163,16 @@ start:
 1:  str  r0, [r2], #4    @ clear bss
     cmp  r2, r3
     blo  1b
-	@ Now we've got rid of the secondary CPUs, set up a stack
-	@ for CPU 0 so we can write most of this in C.
-	ldr     sp, =sec_stacklimit
+    @ Now we've got rid of the secondary CPUs, set up a stack
+    @ for CPU 0 so we can write most of this in C.
+    ldr     sp, =sec_stacklimit
```
Comment (Wonseok) [inline platform-device/cortex_a15x2_arndale/boot.S:168]: Is a CPU1 stack pointer the same as CPU0 stack pointer?

Comment (joohyun0115) [inline platform-device/cortex_a15x2_arndale/boot.S:168]: No. this is a CPU0.
please read ahead(line 131).

**Pull request merged**